amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -163,33 +178,36 @@ Changes the visibility timeout of a specified message in a queue to a new value.
 |Type |String
```

Comment:
Same applies here, string returned by this operation contains ID of the request, shouldn't we mention it here?